incubator/fluentd-cloudwatch/templates/_helpers.tpl

```diff
@@ -2,15 +2,16 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "fluentd-cloudwatch.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{/*
 Create a default fully qualified app name.
 We truncate at 24 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
-{{- define "fluentd-cloudwatch.fullname" -}}
+{{- define "fullname" -}}
```

Comment:
Keep the chart name prefix, more info @ https://github.com/kubernetes/charts/issues/1785